cluster-autoscaler/main.go

```diff
@@ -177,6 +177,7 @@ var (
 	maxPodEvictionTime                  = flag.Duration("max-pod-eviction-time", core.MaxPodEvictionTime, "Maximum time CA tries to evict a pod before giving up.")
 	topologySpreadConstraintScaleFactor = flag.Int("topology-spread-constraint-scale-factor", 0,
 		"If pods with topology spread constraints are present, cap the scale-up size at the number of groups divided by the scale factor.")
+	nodeInstancesCache = flag.Bool("node-instances-cache-enabled", true, "Enable the cloud provider instance cache")
```

Comment:
I'd like `--disable-node-instances-cache` with a default of `false` more.

Reply:
I generally don't like negative options, but having it as an explicit `disable` makes more sense since we're patching upstream behaviour. Will change!